### PR TITLE
Update TerminalSession.java

### PR DESCRIPTION
**Pull request by Auxilus**
Endprocess string modified to check for `~/hushlogout` if exists then exit the terminal without echoing exitDescription

### CODE REVIEW

File: terminal-emulator/src/main/java/com/termux/terminal/TerminalSession.java [hunk 1]
```diff
@@ -115,24 +116,42 @@ public void handleMessage(Message msg) {
...
+            
+            else {
+                if (msg.what == MSG_PROCESS_EXITED) {
+                    File tmpDir = new File("/data/data/com.termux/files/home/.hushlogout");
+                    boolean exists = tmpDir.exists();
+                    if (exists) { 
+                        int exitCode = (Integer) msg.obj;
+                        mShellExitStatus = exitCode;
+                        mShellPid = -1;
+                        mTerminalToProcessIOQueue.close();                         mProcessToTerminalIOQueue.close();
```
Comment (Grimler91) [inline terminal-emulator/src/main/java/com/termux/terminal/TerminalSession.java]: I'm guessing this shouldn't be here twice?

**Comment by Grimler91** [comment]
Wouldn't this sort of undo the very thing that the exit message is suppose to do? 
If a user breaks his environment then there would be no way to launch a failsafe terminal if the terminal exists immediately. 

Maybe it's possible to checkout for /sdcard/hushlogout instead, as @Neo-Oli suggested [here](https://github.com/termux/termux-app/issues/56#issuecomment-192176747)

**Comment by Auxilus** [comment]
Hmm, that sounds like a problem, but I just made temp changes..
Should I add something like warning when used tries `touch ~/hushlogout` ?

**Comment by Auxilus** [comment]
I'll just replace `~/hushlogout` by `/sdcard/hushlogout`

File: terminal-emulator/src/main/java/com/termux/terminal/TerminalSession.java (hunk 1 above)
Comment (Auxilus) [reply terminal-emulator/src/main/java/com/termux/terminal/TerminalSession.java]: Solved..

**Comment by cgarz** [comment]
@Grimler91 True, but the exit message wont protect you from everything anyway. Such as killall com.termux in your bashrc. A way to properly close termux panes programatically is sorely needed.
Many, including myself have said as much on various issues such as [Issue 56](https://github.com/termux/termux-app/issues/56)

**Comment by fornwall** [comment]
Not relevant after #56.

**Pull request closed without merging**